docs/getting-started/first-trigger.md

```diff
@@ -21,18 +24,28 @@
           kind: Service
           name: cloudevents-player
     ```
-==**Expected Output**==
-```{ .bash .no-copy }
-Trigger 'cloudevents-player' successfully created in namespace 'default'.
-```
 
-??? question "What CloudEvents is my `Trigger` listening for?"
-    Since we didn't specify a `--filter` in our `kn` command, our Trigger is listening for any CloudEvents coming into the `Broker`.
+    Once you've created your YAML file (named something like "ce-trigger.yaml"):
```

Comment:
After you've created your YAML file, named something like `ce-trigger.yaml`, apply it by running the command: